TileFormats/Geometry/README.md

```diff
@@ -0,0 +1,110 @@
+# Geometry
+
+## Contributors
+
+* Dan Bagnell, [@bagnell](https://github.com/bagnell)
+* Patrick Cozzi, [@pjcozzi](https://twitter.com/pjcozzi)
+* Sean Lilley, [@lilleyse](https://github.com/lilleyse)
+
+## Overview
+
+The _Geometry_ tile format allows streaming of geometry datasets including boxes, cylinders, ellipsoids, and spheres.
+
+Each box, cylinder, ellipsoid, and sphere is a _feature_ in the 3D Tiles specification language.
+
+In addition to rendering the geometry, the geometry can be used for classifying geometry of other 3D Tiles tilesets.
+
+## Layout
+
+A tile is composed of two sections: a header immediately followed by a body.
+
+## Header
+
+The 28-byte header contains the following fields:
+
+| Field name | Data type | Description |
+| --- | --- | --- |
+| `magic` | 4-byte ANSI string | `"geom"`. This can be used to identify the arraybuffer as a Geometry tile. |
+| `version` | `uint32` | The version of the Geometry Data format. It is currently `1`. |
+| `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. If `featureTableJSONByteLength` is zero, this will also be zero. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. | 
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed, in order, of two fields: `Feature Table` and  `Batch Table`.
+
+Code for reading the header can be found in
+[Geometry3DTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Geometry3DTileContent.js)
+in the Cesium implementation of 3D Tiles.
+
+## Feature Table
+
+Contains values for `geom` semantics used to render features.  The general layout of a Feature Table is described in the [Feature Table specification](../FeatureTable).
+
+The `geom` Feature Table JSON schema is defined in [geom.featureTable.schema.json](../../schema/geom.featureTable.schema.json).
```

Comment:
`geom.featureTable.schema.json` does not exist.

Reply:
Bump

Reply:
Oh actually the file exists but there is a typo in the name: `schema/geom.featureTable.scheme.json` should be `schema/geom.featureTable.schema.json`